src/adapter.js

```diff
@@ -15,9 +15,27 @@
  */
 
 (function(karma, System) {
+    System.config({ baseURL: 'base' });
 
     // Prevent immediately starting tests.
-    window.__karma__.loaded = function() {};
+    window.__karma__.loaded = function() {
```

Comment:
The thought behind this is that you usually want to load your tests after the test environment has been loaded. And since the loaded method was anyway overwritten to prevent the immediate start of the tests, why not use this method to load the tests then, since this is the point from where on the tests should run.

It has nothing to do with the baseURL configuration but is rather just a small improvement.